jep/1/README.adoc

```diff
@@ -0,0 +1,754 @@
+
+.JEP Workflow
+image::workflow.png[JEP Workflow]
+
+When a JEP is Accepted, Rejected or Withdrawn,
+the "Resolution" header must be added with a link to the relevant post
+in the jenkinsci-dev@ mailing list archives.
+
+
+=== What belongs in a successful JEP?
+
+Each JEP should have the following parts:
+
+. **Metadata** - table containing metadata about the JEP, including the JEP
+  number, a short descriptive title, the names, and optionally the contact info
+  for each author, etc.
+. **Abstract** - short (200 word) description of the technical issue
+  being addressed.
+. **Specification** - The technical specification should describe the
+  syntax and semantics of any new feature. The specification should be
+  sufficiently detailed to allow new or existing Jenkins developers to
+  reasonably understand the scope/impact of an implementation.
+. **Motivation** - The motivation is critical for JEPs that want to
+  change Jenkins itself. It should clearly explain why the
+  existing code base is inadequate to address the
+  problem that the JEP solves. JEP submissions without sufficient
+  motivation may be rejected outright.
+. **Reasoning** - The reasoning describes why particular design
+  decisions were made. It should describe alternate designs that
+  were considered and related work, e.g. how the feature is supported
+  in other languages.
++
+The reasoning should provide evidence of consensus within the
+community and discuss important objections or concerns raised
+during discussion.
+
+. **Backwards Compatibility** - All JEPs that introduce backwards
+  incompatibilities must include a section describing these
+  incompatibilities and their severity. The JEP must explain how the
+  author proposes to deal with these incompatibilities. JEP submissions that do
+  not adequately discuss backwards compatibility, when such discussion is
+  required, may be rejected outright.
+. **Reference Implementation** -- The reference implementation must be
+  completed before any JEP is given status "Final", but it need not
+  be completed before the JEP is accepted. While there is merit
+  to the approach of reaching consensus on the specification and
+  rationale before writing code, the principle of "rough consensus
+  and running code" is still useful when it comes to resolving many
+  discussions of API details.
+. **References** -- When moving a JEP from a Draft to Accepted or Final state,
+  the references section should be updated to include links to the pull requests
+  and mailing list discussions which were involved in the process. The JEP
+  should self-document the process in which it was developed.
+
+The final implementation must include test code and documentation
+appropriate for either the Jenkins user or developer documentation.
+
+
+==== JEP Formats and Templates
+
+JEPs are UTF-8 encoded text files using the
+link:https://asciidoctor.org[AsciiDoc] format.
+AsciiDoc allows for rich markup that is still quite easy to read,
```

Comment:
Tables are bad in any markup language, see this doc :(
For design docs I use many tables, but it's something to deal with